raiden/settings.py

```diff
@@ -58,7 +58,8 @@
 DEFAULT_MEDIATION_PROPORTIONAL_IMBALANCE_FEE = ProportionalFeeAmount(
     3000  # 0.3% in parts per million
 )
-DEFAULT_MEDIATION_FEE_MARGIN: float = 0.05
+DEFAULT_MEDIATION_FEE_MARGIN: float = 0.03
+PAYMENT_AMOUNT_BASED_FEE_MARGIN: float = 0.0025
```

Comment:
Should this be prefixed with `DEFAULT_`?

Reply:
There's no way to override this value at the moment, so I wouldn't call it default.